### PR TITLE
scripts: EternalShell support for gceworker

### PR DESCRIPTION
**Pull request by tbg**
`mosh` is nice, but it messes with the scrollback capabilities of the
terminal, which is unfortunate when the shell is used for long-running
commands whose output matters.

Baking in workarounds for mosh into gceworker seemed more complex than also
offering EternalShell (et), which works in more predictable ways.
`et` does not offer "typing prediction" and thus feels a little less
snappy, but our gceworkers are typically not high-latency.

Also fixes a typo in the .bashrc_bootstrap file. Not sure how this worked
in the past.

Release note: None

### CODE REVIEW

File: scripts/gceworker.sh [hunk 1]
```diff
@@ -30,12 +30,16 @@ case "${cmd}" in
...
     gcloud compute firewall-rules create "${NAME}-mosh" --allow udp:60000-61000
+    gcloud compute firewall-rules create "${NAME}-et" --allow tcp:2022
 
     # Retry while vm and sshd to start up.
     retry gcloud compute ssh "${NAME}" --command=true
 
     gcloud compute copy-files "build/bootstrap" "${NAME}:bootstrap"
     gcloud compute ssh "${NAME}" --ssh-flag="-A" --command="./bootstrap/bootstrap-debian.sh"
+    # This was not put in bootstrap-debian.sh because it's for Ubuntu (the Debian version of it
+    # was broken).
```
Comment (benesch) [inline scripts/gceworker.sh]: Where is bootstrap-debian used to bootstrap something that's not actually Ubuntu? Maybe we can just rename that thing bootstrap-ubuntu?

**Review by benesch — APPROVED** [review]

File: scripts/gceworker.sh [hunk 2]
```diff
@@ -30,12 +30,16 @@ case "${cmd}" in
            --boot-disk-device-name "${NAME}" \
            --scopes "default,cloud-platform"
     gcloud compute firewall-rules create "${NAME}-mosh" --allow udp:60000-61000
+    gcloud compute firewall-rules create "${NAME}-et" --allow tcp:2022
```
[the remark this answers is not in the document]
Comment (tschottdorf) [reply scripts/gceworker.sh]: I can add a prompt during create. How's that?
I should also really really make these firewall rules specific to the gceworker. I think they are global right now.

[the remark this answers is not in the document]
Comment (tschottdorf) [reply scripts/gceworker.sh]:
```
Host is up (0.13s latency).
Not shown: 997 filtered ports
PORT     STATE  SERVICE
22/tcp   open   ssh
2022/tcp open   down
3389/tcp closed ms-wbt-server
```

**Comment by tschottdorf** [comment]
I was able to make https://unix.stackexchange.com/questions/145181/use-mosh-without-giving-up-local-scrollback-history work (though not in this PR). Thoughts about using that approach instead? EternalTerminal really feels a lot better, but it's at the point where it doesn't matter, and opening that port is indeed sketchy.

**Comment by tschottdorf** [comment]
Still think mosh doesn't work that well out of the box, but I'm not going to have bandwidth to improve on this here.

**Pull request closed without merging**